### ISSUE TITLE
Linux doesnt work when the messages contains `<>`

### ISSUE THREAD

**Issue by xMAC94x** [balthild/native-dialog-rs#10]
```rust

    MessageAlert {
        title: "FOOBAR",
        //somehow `<` and `>` are invalid characters and cause the msg to get replaced
        // by some generic text thus i replace them
        text: "Doenst <work>",
        typ: MessageType::Error,
    }
        .show()
        .unwrap();
```

displays another message. it shows : "An error has occured"
tested on arch with i3

**Comment by balthild** [comment]
Thanks for reporting. The cause is that zenity and kdialog supports html syntax, and `<>&` are special chars. When there's syntax erroes in messages, kdialog will treat them as pure texts but zenity will fail. I should escape these characters before calling zenity.

**Comment by balthild** [comment]
Fixed in v0.4.3
